ARTICLE-2.markdown

```diff
@@ -2,8 +2,8 @@
 
 ### Section 2.1  Voluntary Withdrawal
 
-A Member shall have the right to resign from the Corporation, effective
-immediately without any action on the part of the Corporation.
+A Member shall have the right to resign from the Cooperative, effective
+immediately without any action on the part of the Cooperative.
 
```

Comment:
I have a feeling not all will agree to this. In fact, I'm 100% sure of it :P  If it is a matter of cash flow, then our cash buffer needs to be raised. Withholding a member's initial capital contribution and patronage would be very unusual for a cooperative, and I think it would discourage potential members, as it effectively punishes them for leaving.